tools/webhook-endpoint/.gitignore

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
"webhook-endpoint" is a little non-descript of a directory name for a tool perhaps "webhook-data-ingest"?
Also you need an entry in the README at the root of the repo, please.

Reply:
Added to README
Renamed the directory to `webhook-ingestion-pipeline`